src/utils/long.util.ts

```diff
@@ -1,17 +1,17 @@
 import Long from 'long';
 
+const POSTGRES_MAX_BIGINT = BigInt('9223372036854775807');
+const POSTGRES_MIN_BIGINT = BigInt('-9223372036854775808');
```

Comment:
create constant.ts file and move this to it.

Reply:
@Ko1103 Never use in other modules, still should move it?

Reply:
ok